<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>582</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1033</width>
    <height>513</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9000</width>
    <height>9000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tbl_main">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>971</width>
      <height>291</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Region</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true" extracomment="Natural Population Growth">Nat Pop Grow</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Birth Rate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Death Rate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gen Dem Rate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Urbanization</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_view_all">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести полную таблицу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_view_part">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести часть таблички</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_region">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите фильтр для поиска:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_column">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите номер коллонки(3 - 7):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_column_count">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_txt_main">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_max">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_txt_min">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_min">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_med">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_txt_med">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Med:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect_file">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключить файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Текущий файл: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file_name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NONE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_region">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Choose region</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cb_colunm">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Choose column</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
